### PR TITLE
Add login checking logic

### PR DESCRIPTION
**Pull request by bananapizzza**

### CODE REVIEW

File: src/actions/actions_login.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import fetch from "cross-fetch";
+
+export const REQUEST_LOGIN = 'REQUEST_LOGIN';
```
Comment (xaveng) [inline src/actions/actions_login.js]: Let's add a comment for exported entities.

File: src/actions/actions_login.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    let isLoginFail = false;
+    fetch("http://localhost:5000/check_login_info", {
+        method: "POST",
+        headers: {
+            'Content-type': 'application/json',
+        },
+        body: JSON.stringify(values)
+    }).then(result => {
+        if (result.status >= 400) {
+            isLoginFail = true;
```
Comment (xaveng) [inline src/actions/actions_login.js]: we can throw an exception and catch it from next

File: src/actions/actions_login.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        }
+        return result.json();
```
Comment (xaveng) [inline src/actions/actions_login.js]: Use can do like this:
```
}).then(result => {
        if (result.status >= 400) {
            throw exception();
        }
        const resultJson = result.json();
        dispatch(loginSuccess(resultJson .username));
}).catch(e =>  dispatch(loginFail(json)));
```

File: src/actions/actions_login.js (hunk 1 above)
Comment (bananapizzza) [reply src/actions/actions_login.js]: Okay

File: src/actions/actions_login.js (hunk 3 above)
Comment (bananapizzza) [reply src/actions/actions_login.js]: Changed to use async/await.

**Pull request merged**